Model/System/Config.php

```diff
@@ -116,6 +119,42 @@ public function getShippingCostUrl()
      */
     public function isDisplayDeliveryTimeOnProductListing()
     {
-        return (bool)$this->scopeConfig->getValue('catalog/frontend/display_delivery_time', ScopeInterface::SCOPE_STORE);
+        return (bool)$this->scopeConfig->getValue(
+            'catalog/frontend/display_delivery_time',
+            ScopeInterface::SCOPE_STORE
+        );
+    }
+
+    /**
+     * Checks if dynamic shipping tax class is active.
+     *
+     * It means shop will take product tax class with highest rate from current cart product items
+     * and then use it as shipping tax class for this cart/quote during checkout.
+     *
+     * @return bool
+     */
+    public function isDynamicShippingTaxClassActive()
+    {
+        return (bool)$this->scopeConfig->getValue(
```

Comment:
Can we use `isSetFlag`?

Reply:
any thinking behind this? we should probably change other methods as well to match that

Reply:
Since `isSetFlag` does exactly what we want, I think it just makes sense to use it. I did not check the old code, but IMHO, we should use it where a `bool` is retrieved from the config. If you would not like to change this, fine for me.